### ISSUE TITLE
[Filebeat} S3 Invalid Gzip Header

### ISSUE THREAD

**Issue by nathankodilla** [elastic/beats#18988]
All s3 files we are attempting to read have the following error:
```
ERROR	[s3]	s3/input.go:463	gzip.NewReader failed: gzip: invalid header
ERROR	[s3]	s3/input.go:395	createEventsFromS3Info failed for thefile.txt.gz: gzip.NewReader failed: gzip: invalid header
```

Metadata:
![image](https://user-images.githubusercontent.com/22841858/83825348-1af16b80-a6a7-11ea-81d1-4bb825af8294.png)

These files were previously read in by older versions fine. Possibly related to https://github.com/elastic/beats/issues/16374

Version 7.7.1

**Comment by kaiyan-sheng** [comment]
@ycombinator I think this should be fixed by https://github.com/elastic/beats/pull/18764?

**Comment by kushagra-io** [comment]
@kaiyan-sheng Has there been an update to this issue yet? We're experiencing the same issue as well on filebeat version 7.13.3

**Comment by kaiyan-sheng** [comment]
This should be fixed by #18764.
